Tests/UI/BillingForm/ViewModel/PaymentViewModelTests.swift

```diff
@@ -626,7 +660,7 @@ class PaymentViewModelTests: XCTestCase {
         BillingForm(name: "John", address: Address(addressLine1: "Kong Drive", addressLine2: "Blister", city: "Dreamland", state: nil, zip: "DR38ML", country: Country.allAvailable.first), phone: nil)
     }
     
-    private func makeBillingFormStyle(isCardholderMandatory: Bool = false,
+    private func makePaymentFormStyle(isCardholderMandatory: Bool = false,
```

Comment:
What is the purpose of this renaming ? from makeBillingFormStyle to makePaymentFormStyle

Reply:
The output was a PaymentFormStyle, not a Billing. So the function name was not consistent with the object being created.